Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -329,15 +334,20 @@ private void showBottomSheet() {
 
     private void showCardDetailsFragment(final String cardNumber) {
         if (shouldAddFragment(CARD_DETAILS_TAG)) {
-            dropInClient.getConfiguration((configuration, error) -> {
-                if (configuration != null) {
-                    // TODO: implement dropInClient.hasAuthType(AuthType.TOKENIZATION_KEY)
-                    boolean hasTokenizationKeyAuth =
-                            Authorization.isTokenizationKey(dropInClient.getAuthorization().toString());
-
-                    CardDetailsFragment cardDetailsFragment = CardDetailsFragment.from(
-                            dropInRequest, cardNumber, configuration, hasTokenizationKeyAuth);
-                    replaceExistingFragment(cardDetailsFragment, CARD_DETAILS_TAG);
+            dropInClient.getAuthorization(new AuthorizationCallback() {
+                @Override
+                public void onAuthorizationResult(@Nullable Authorization authorization, @Nullable Exception authorizationError) {
+                    dropInClient.getConfiguration((configuration, configurationError) -> {
+                        // TODO: determine UX for scenario where authorization or configuration could not be fetched
```

Comment:
We could probably do something like what we did before for `InvalidAuthorization`. Maybe error message like "There was a problem fetching authorization, or the authorization was invalid"?

Reply:
Good call! I can finish the activity with the authorization error.